build_tools/buildkite/cmake/linux/x86-64/benchmark.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+steps:
```

Comment:
this path (`build_tools/buildkite/cmake/linux/x86-64/benchmark.yml`) is fine for now, but we might want to refactor to `build_tools/buildkite/pipelines/benchmarks/linux_x86-64.yml` and `build_tools/buildkite/pipelines/benchmarks/android_arm64-v8a.yml` (or similar) now that we'll be adding more buildkite pipelines

Reply:
Same as the comment above. I haven't finished the details of the new benchmark pipeline. So I'd prefer to keep the current format and hope it won't get too worse in the short term.